packages/sirius-web/backend/sirius-web-graphql/src/main/java/org/eclipse/sirius/web/graphql/datafetchers/editingcontext/EditingContextRepresentationDataFetcher.java

```diff
@@ -101,14 +97,7 @@ public DataFetcherResult<RepresentationMetadata> get(DataFetchingEnvironment env
     }
 
     private RepresentationMetadata toRepresentationMetadata(IRepresentation representation) {
-        // @formatter:off
-        String targetObjectId = Optional.of(representation)
-                .filter(ISemanticRepresentation.class::isInstance)
-                .map(ISemanticRepresentation.class::cast)
-                .map(ISemanticRepresentation::getTargetObjectId)
-                .orElse(null);
-        // @formatter:on
-        return new RepresentationMetadata(representation.getId(), representation.getKind(), representation.getLabel(), representation.getDescriptionId(), targetObjectId);
+        return new RepresentationMetadata(representation.getId(), representation.getKind(), representation.getLabel(), representation.getDescriptionId());
```

Comment:
You could get rid of this method since it has no real purpose anymore and inline the code.